com.github.textosaurus.json

```diff
@@ -0,0 +1,46 @@
+{
+    "build-options": {
```

Comment:
This is a very strange layout. Please be consistent with the order other manifest use.

Reply:
Is order somehow defined in Flatpak docs? I can reorder it I guess, just autoformatted.

Reply:
No it isn't defined. Alphabetical just doesn't make sense.